src/lib/Microsoft.Health.Logger/Telemetry/Exceptions/ExceptionExtensionMethods.cs

```diff
@@ -0,0 +1,28 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
+
+namespace Microsoft.Health.Logging.Telemetry.Exceptions
+{
+    public static class ExceptionExtensionMethods
```

Comment:
nit: should the name indicate something to do with logging/telemetry instead or are we keeping it open for any kind of extension on exceptions? Also, 'Methods' be removed?

Reply:
> should the name indicate something to do with logging/telemetry instead or are we keeping it open for any kind of extension on exceptions?

I separated it for exceptions because the pattern of Telemetry/Exceptions and Telemetry/Metrics seems to lie throughout the projects. The new extension method is also specific to exceptions.

---

> Also, 'Methods' be removed?

I considered it, but I kept it to parallel `MetricExtensionMethods`..